### ISSUE TITLE
Cupcake website - new Homepage image explorations

### ISSUE THREAD

**Issue by JennySanchez** [carbon-design-system/ibm-dotcom-library-website#248]
> As an IBM.com library website creator

> I need to explore different ways to present the content on the homepage

> so that our adopters can not only find information easily but also visually engaging with the layout.

#### Task

@jacobottesen-dgc As discussed, let's timebox ourselves and work on ~~new layouts~~ new image treatments for the homepage.

- The goal is to make the Homepage better than what we currently have.
- The current Homepage can be viewed [here](https://ibm-dotcom-library.netlify.com/).
- the latest Homepage Sketch work file can be found [here](https://ibm.box.com/s/gs0apv3kyopoiy72o3gphefbdm0op8xx).

#### Acceptance criteria
- [ ] Review designs with Wonil and define the next step.

**Comment by wonilsuhibm** [comment]
@JennySanchez I suggest making the lead space a bit shorter.

**Comment by wonilsuhibm** [comment]
@JennySanchez I added some more [design explorations](https://ibm.box.com/s/wecud78ka6a4b49xl8q201iq2vovsuo0) and a rough [video cut](https://ibm.box.com/s/ff9aee80fogw1lxaabm535gegn7u2djp).

@jeffchew Any chance we can play with the video in the home page lead space? I know some coding, maybe you can show me how to plug-in a video in the lead space so that I can play with different versions without having to bug you all the time.

**Comment by jeffchew** [comment]
> @JennySanchez I added some more [design explorations](https://ibm.box.com/s/wecud78ka6a4b49xl8q201iq2vovsuo0) and a rough [video cut](https://ibm.box.com/s/ff9aee80fogw1lxaabm535gegn7u2djp).
> 
> @jeffchew Any chance we can play with the video in the home page lead space? I know some coding, maybe you can show me how to plug-in a video in the lead space so that I can play with different versions without having to bug you all the time.

@wonilsuhibm the `HomepageVideo` component is specific to the carbon website, and will need to be pulled in from there.

**Comment by wonilsuhibm** [comment]
@jeffchew Ah okay. So we should go with a still image for now, and add a new story for video. Thank you!

**Comment by wonilsuhibm** [comment]
Here is a still image suggestion until we figure out the video part:
<img width="1355" alt="Screen Shot 2019-12-31 at 12 01 41 PM" src="https://user-images.githubusercontent.com/30183447/71628367-94030900-2bc5-11ea-9c5e-97f35f56236b.png">

**Comment by JennySanchez** [comment]
@wonilsuhibm I like the idea of using video but we need to get the requirements of how the video should be formatted or created from Jeannie (maybe)?

**Comment by JennySanchez** [comment]
@RobertaJHahn FYI - The milestone is updated from Sprint 24 to Sprint 20-1.

**Comment by wonilsuhibm** [comment]
The latest suggestion from Brand is to update the current image as an MVP and work on a video version.

Things to cover:
- darker overlay
- smaller components (zoomed-out view)
- other visual treatments...

**Comment by JennySanchez** [comment]
**Task updates:**
- Shared two homepage hero image options with Mike for feedback. We are good to go with either image.
- Shared two release schedule hero image options with Mike for feedback and waiting on him to get back to me on this.

![Hero Image 04](https://user-images.githubusercontent.com/45213839/72109045-8acb1800-32fa-11ea-9771-a02d0aeb8e87.jpg)
![Hero Image 04 Copy](https://user-images.githubusercontent.com/45213839/72109046-8bfc4500-32fa-11ea-8c39-c8b86cefe9a8.jpg)

**Comment by JennySanchez** [comment]
@wonilsuhibm Are you be ok to move forward with the image below?
Between the two, I like this one better bc it provides depth with a good amount of the detail.
![Screen Shot 2020-01-09 at 6 00 11 PM](https://user-images.githubusercontent.com/45213839/72114836-7727ad80-330a-11ea-9275-3aae719a8c79.png)

**Comment by wonilsuhibm** [comment]
@JennySanchez Thank you! Looks awesome. Let's close this issue.
